en/08_Changelogs/5.2.0.md

```diff
@@ -405,6 +405,7 @@ A [`LogicException`](https://www.php.net/manual/en/class.logicexception.php) is
 - The assets admin section now respects the [`Upload_Validator.default_max_file_size`](api:SilverStripe\Assets\Upload_Validator->default_max_file_size) configuration if [`AssetAdmin.max_upload_size`](api:SilverStripe\AssetAdmin\Controller\AssetAdmin->max_upload_size) has not been explicitly set for your project. Previously, asset admin would ignore `Upload_Validator.default_max_file_size` and just use the PHP `upload_max_filesize` ini configuration by default.
 - Redirector pages with a link to a missing or unpublished page/file will now return a 404 response. You can revert to the previous behaviour by setting the [`RedirectorPageController.missing_redirect_is_404`](api:SilverStripe\CMS\Model\RedirectorPageController->missing_redirect_is_404) configuration property to `false`.
 - Developers can now opt in to including a comment on all ORM database queries indicating where they were executed from. See [debugging database queries](/developer_guides/debugging/#debugging-database-queries) for details.
+- The [Composer runtime api](https://getcomposer.org/doc/07-runtime.md#installed-versions) is now being used in the VersionProvider class instead of relying on the composer.lock file.
```

Comment:
```suggestion
- The [Composer runtime api](https://getcomposer.org/doc/07-runtime.md#installed-versions) is now being used in the [`VersionProvider`](api:SilverStripe\Core\Manifest\VersionProvider) class instead of relying on the `composer.lock` file.
```